builds/misc/templates/build-rocksdb.yaml

```diff
@@ -5,27 +5,50 @@ stages:
   - stage: BuildRocksDB
 ################################################################################
     displayName: Build RocksDB
-    pool:
-      name: $(pool.linux.name)
-      demands:
-        - ImageOverride -equals agent-aziotedge-ubuntu-20.04-docker
     dependsOn: ${{parameters.dependency}}
     jobs: 
     - job: BuildRocksDB
       displayName: Build Rocks DB
       strategy:
         matrix:
           RocksDbAmd64:
+            pool_name: $(pool.linux.name)
+            agent_image: agent-aziotedge-ubuntu-20.04-docker
             arch: amd64
             postfix: amd64
           RocksDbArm32:
+            pool_name: Azure-IoT-Edge-1ES-Hosted-Linux-Arm64
+            agent_image: agent-aziotedge-ubuntu-20.04-arm64
             arch: arm32v7
             postfix: armhf
           RocksDbArm64:
+            pool_name: Azure-IoT-Edge-1ES-Hosted-Linux-Arm64
+            agent_image: agent-aziotedge-ubuntu-20.04-arm64
             arch: arm64v8
             postfix: arm64
+      pool:
+          name: $(pool_name)
+          demands:
+          - ImageOverride -equals $(agent_image)
       steps:
-      - script: scripts/linux/buildRocksDb.sh --output-dir $(Build.ArtifactStagingDirectory) --postfix $(postfix) --build-number $(Build.BuildNumber) --arch $(arch)
```

Comment:
Looks like buildRocksDb.sh still exists and we use it in buildLocalEdgeHub.sh. Should we use it here too?

Reply:
We still call it...just a little further down. See line 50.